lib/auth/webauthn/login.go

```diff
@@ -67,13 +67,18 @@ type loginFlow struct {
 	sessionData sessionIdentity
 }
 
-func (f *loginFlow) begin(ctx context.Context, user string, passwordless bool) (*wantypes.CredentialAssertion, error) {
-	if user == "" && !passwordless {
+func (f *loginFlow) begin(ctx context.Context, user string, scope wanpb.ChallengeScope, allowReuse bool) (*wantypes.CredentialAssertion, error) {
+	if allowReuse && scope != wanpb.ChallengeScope_CHALLENGE_SCOPE_ADMIN_ACTION {
+		return nil, trace.BadParameter("mfa challenges with scope %v cannot allow reuse", scope)
```

Comment:
```suggestion
		return nil, trace.BadParameter("mfa challenges with scope %s cannot allow reuse", scope)
```

so it prints the name, not the number.